starknet_py/net/full_node_client.py

```diff
@@ -812,7 +813,7 @@ def _get_raw_block_identifier(
     return "pending"
 
 
-def _create_broadcasted_txn(transaction: AccountTransaction) -> dict:
+def _create_broadcasted_txn_prev(transaction: AccountTransaction) -> dict:
```

Comment:
what does `prev` stand for in here? (this potentially is a nitpick)

Reply:
prev version of `_create_broadcasted_txn` it will be remove before merge to development